README.md

```diff
@@ -1,2 +1,64 @@
-# eslint-package-boundaries
-Makes sure that all the imports and exports happen trough index files
+# eslint-plugin
+
+![npm](https://img.shields.io/npm/v/@lokalise/eslint-plugin)
+![Downloads total](https://img.shields.io/npm/dt/@lokalise/eslint-plugin)
+
```

Comment:
Would be great to give a brief explanation of what those rules cover: are they for FE, BE, do they include anything framework-specific? Are they completely standalone or are derived from other popular ruleset?

Reply:
Maybe @artursvonda (I think he first built this) could help me elaborate further on this over this document -> https://github.com/lokalise/eslint-plugin/blob/d7f8cc3733800ea642a563838c3b529706547e18/docs/rules/package-boundary.md